aclogview/CM_Vendor.cs

```diff
@@ -18,13 +19,20 @@ public class CM_Vendor : MessageProcessor {
                     message.contributeToTreeView(outputTreeView);
                     break;
                 }
-            case PacketOpcode.Evt_Vendor__Sell_ID: {
+            case PacketOpcode.Evt_Vendor__Sell_ID:
+                {
                     Sell message = Sell.read(messageDataReader);
                     message.contributeToTreeView(outputTreeView);
                     break;
                 }
+            case PacketOpcode.VENDOR_INFO_EVENT:
+                {
+                    gmVendorUI message = gmVendorUI.read(messageDataReader);
+                    message.contributeToTreeView(outputTreeView);
+                    break;
+                }
```

Comment:
This block should be kept under the TODO for PacketOpcode.Evt_Vendor__RequestVendorInfo_ID below, to maintain the order of packet opcodes